tests/lagrangian_detectors_oscillating_rotation/plotscript.py

```diff
@@ -0,0 +1,21 @@
+from scipy import *
```

Comment:
Add a comment to say what this script does. Doesn't appear to actually plot anything, despite name.